conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: graphistry-dev
+  version: "0.25.2"
+
+source:
+  path: ./
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - graphistry_dev.graphistry_core
+    - graphistry_dev.graphistry-umap
+    - graphistry_dev.graphistry-ai
+
+test:
+  requires:
+    - flake8
+    - mock
+    - mypy
+    - pytest
+    - pandas-stubs
+    - types-requests
+
+outputs:
+  - name: graphistry_dev.graphistry_core
+    requirements:
```

Comment:
is there anything like `outputs.*.description` where they can point to the main package?

Reply:
We can put instructions in the main description and it will be used for all outputs

Reply:
You can check [here ](https://anaconda.org/pygraphistry_dev/graphistry.graphistry-core) to see how the description looks like right now